docs/public-networks/concepts/parallel-transaction-execution.md

```diff
@@ -0,0 +1,194 @@
+in Besu storage.
+
+The following flowchart outlines how Besu detects conflicts and imports transactions into the block:
+
+<p align="center">
+
+```mermaid
+graph TD;
+  A(Start block import) --> B(Fetch block's touched addresses);
+  B --> C{{For each transaction}};
+  C -->|Next transaction| D(Fetch transaction's touched addresses);
+  D --> E{{Compare addresses}};
+  E -->|Conflict detected| F(Replay transaction using cached data);
+  E -->|No conflict| G(Apply transaction result directly – no replay);
+  F --> H{{Attempt to read from cache}};
+  H -->|Data found in cache| I(Continue replay using cached data);
+  H -->|Data not found in cache| J(Fetch data from disk);
+  I --> K(Transaction replay complete);
+  J --> K;
+  K --> L(Apply transaction changes);
+  G --> L;
+  L --> M{{More transactions?}};
+  M -->|Yes| C;
+  M -->|No| N(End block import);
+```
+
+</p>
+
+Besu takes what the accumulator tracks at the block and transaction level, compares the
+transaction's list of touched addresses to the block's list, and checks for conflicts.
+In particular:
+
+1. Besu identifies conflicts by checking whether a transaction has interacted with accounts modified
+   by the block (that is, modified by previous transactions).
+2. If a conflict is detected, Besu replays the transaction using cached data or data fetched from disk.
+3. Each time a transaction is added to the block, Besu incorporates the transaction's tracked list
+   into the block's list.
+
+:::info Note
+The following are excluded from the conflict check:
+
+- Unchanged accounts read by the block.
+- Rewards given to the validator coinbase address at the end of each transaction.
+  If these were considered, every transaction would conflict with the coinbase address.
+  Besu identifies this address as a conflict only if it is accessed for reasons other than receiving
+  rewards at the transaction's conclusion.
+:::
+
+The following flowchart outlines how Besu maintains the lists of tracked addresses:
+
+<p align="center">
+
+```mermaid
+graph TD;
+  A(Start) --> B(Fetch block's touched addresses);
+  B --> C{{Check each address}};
+  C -->|Unchanged| D(Mark as read);
+  C -->|Modified| E(Add to block's tracked addresses);
+  D --> F{{Next address}};
+  E --> F;
+  F -->|More addresses?| C;
+  F -->|No more| G(Fetch transaction's touched addresses);
+  G --> H{{For each transaction address}};
+  H -->|From, sender, etc.| I(Add to transaction's tracked addresses);
+  I --> J{{Next address}};
+  J -->|More addresses?| H;
+  J -->|No more| K{{Compare block and transaction addresses}};
+  K -->|Conflict detected| L(Conflict is detected);
+  K -->|No conflict| M(Proceed with transaction);
+  L --> N(End);
+  M --> N;
+```
+
+</p>
+
+Besu's conflict detection strategy is intentionally simple to minimize edge cases.
+With this approach to parallel transaction execution,
+[approximately 40% of transactions do not require replay](#metrics).
```

Comment:
Only thing worth noting here is that if we do refine the conflict detection strategy in the future this value may need to change.